ext/opentelemetry-ext-jaeger/tests/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2019, OpenTelemetry Authors
```

Comment:
I think we don't add license boilerplate to otherwise completely empty files usually.